charts/steward/templates/deployment-run-controller.yaml

```diff
@@ -33,13 +33,12 @@ spec:
         image: {{ printf "%s:%s" .repository .tag | quote }}
         imagePullPolicy: {{ .pullPolicy | quote }}
         {{- end }}
-        args: [
-        "-qps={{ .Values.runController.args.qps }}",
-        "-burst={{ .Values.runController.args.burst }}",
+        args:
+        - {{ printf "-qps=%d" ( .Values.runController.args.qps | int ) | quote }}
+        - {{ printf "-burst=%d" ( .Values.runController.args.burst | int ) | quote }}
         {{- if .Values.runController.args.logVerbosity }}
-        "-v", {{ .Values.runController.args.logVerbosity | quote }},
+        - {{ printf "-v=%d" ( .Values.runController.args.logVerbosity | int ) | quote }}
```

Comment:
I'm not sure if it works with one parameter `-v=<verbosity>`

Reply:
It's mentioned like this in the [K8s logging conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions).

`klog.InitFlags()` is based on the `flag` package. `flag.Parse()` allows `-flag=value` (see [docs](https://golang.org/pkg/flag/#hdr-Command_line_flag_syntax)).

So, I'm confident it works.

Reply:
Tested, works.